base_product_merge/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Base Products Merge",
+    "summary": "Merge duplicate products",
+    "version": "15.0.1.0.0",
+    "license": "LGPL-3",
```

Comment:
Now I see that openupgrade merge records tool is AGPL: https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade_merge_records.py

Everything depending on AGPL must be AGPL